tests/integration/targets/setup_sqlserver/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install Powershell modules
+  ansible.builtin.command: >
+    pwsh -Command "
+      if (-not(Get-Module -FullyQualifiedName @{ModuleName='dbatools';ModuleVersion='{{ dbatools_min_version }}'} -ListAvailable)) {
+        Install-Module dbatools -MinimumVersion {{ dbatools_min_version }} -Force
+      }"
```

Comment:
Eventually I think we'll have to re-think this. 

It's nice to avoid dependencies on other collections for tests, but I think at some point we'll need to depend on the Windows collections (for tests) anyway; at that point we can either use `community.windows.win_psmodule` or `ansible.windows.win_shell` with `pwsh_runner` (I have my doubts the former will work as written, but it's possible).

Reply:
Agreed, windows testing down the road will probably be helpful for more Windows-based modules, but I think currently the existing collection is completely core compatible